install/delete_model.sh

```diff
@@ -1,76 +0,0 @@
-#!/bin/bash
-
-if [ $# -lt 1 ]; then
-    echo "Usage: $0 \"username/modelname\"."
```

Comment:
I guess we are deleting this because we have already an endpoint doing this stuff?